### PR TITLE
Add postgres request time statistics

### PR DESCRIPTION
**Pull request by troopa81**
Integrate postgres request execution time in every GetMap and GetCapabilities request statistics that have set the property PROVIDER to POSTGRES.

### CODE REVIEW

File: graffiti/config.py [hunk 1]
```diff
@@ -37,6 +46,8 @@ def __init__(self, cfg, basedir, logdir, precision=2):
         self.name = cfg['NAME']
         self.logdir = logdir
         self.precision = precision
+        self.provider = cfg.get('PROVIDER', None)
```
Comment (elemoine) [inline graffiti/config.py]: Minor, but `None` is not necessary here.

File: requirements.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ pyaml
 pygal
 requests
 tqdm
+psycopg2
```
Comment (elemoine) [inline requirements.txt:6]: Isn't `pyscopg2-binary` the package that should be used nowadays?

**Comment by elemoine** [comment]
Just provided a few comments, really as an excuse to learn about Graffiti and your changes :-)

File: requirements.txt (hunk 2 above)
Comment (troopa81) [reply requirements.txt:6]: Did you have any documentation that says to prefer `psycopg2-binary` over `psycopg2`. I can find only debate with unclear (to me) conclusion.

File: graffiti/config.py (hunk 1 above)
Comment (troopa81) [reply graffiti/config.py]: Corrected 8bb1458

**Comment by pblottiere** [comment]
Hi @troopa81,

I tried to run graffiti with QGIS Server but I have the next issue:

```
Requests:   0%|                                                                                                                                         | 0/1 [00:00<?, ?it/s]
Traceback (most recent call last):                                                                                                                                            
  File "./graffiti.py", line 52, in <module>                                                                                                                                  
    req.run()
  File "/home/pblottiere/tmp/QGIS-Server-PerfSuite/scenarios/graffiti/graffiti/request.py", line 145, in run
    self.after_request(log, host)
TypeError: after_request() takes 2 positional arguments but 3 were given
```

Can you take a look please?

**Comment by troopa81** [comment]
@pblottiere corrected in d933a7e

**Comment by troopa81** [comment]
@pblottiere Is this ok for merge?

**Comment by pblottiere** [comment]
Thanks @troopa81 :+1:!

**Pull request merged**